pkg/ext/ext.go

```diff
@@ -2,16 +2,16 @@ package ext
 
 import (
 	"github.com/augmentable-dev/flite/internal/http"
-	"github.com/augmentable-dev/flite/internal/lines"
 	"github.com/augmentable-dev/flite/internal/readfile"
+	split "github.com/augmentable-dev/flite/internal/split"
```

Comment:
if the package is renamed to `split` above you shouldn't need this alias here (can just have the import string)